Dockerfile

```diff
@@ -0,0 +1,36 @@
+# Use rust image as the builder base
+FROM rust:1.66.0-bullseye AS builder
+
+# Set the build profile to be release
+ARG BUILD_PROFILE=release
+ENV BUILD_PROFILE $BUILD_PROFILE
+
+# Update and install dependencies
+RUN apt-get update && apt-get -y upgrade && apt-get install -y libclang-dev pkg-config
+
+# Copy base folder into docker context
+COPY . reth
+
+# Build reth
+RUN cd reth && cargo build --all --profile release
+
+# Use ubuntu as the release image
+FROM ubuntu:22.04
+
+# Update dependencies and add ethereum ppa
+RUN apt-get update && apt-get -y upgrade && apt-get install -y --no-install-recommends \
+  libssl-dev \
+  ca-certificates \
+  gnupg2 \
+  software-properties-common && apt-key update \
+  && add-apt-repository -y ppa:ethereum/ethereum
+
+# Install Geth as it's a dependency for reth
+RUN apt-get update && apt-get install -y ethereum \
+  && apt-get clean \
+  && rm -rf /var/lib/apt/lists/*
```

Comment:
These should not be required..geth is def not a dependency to run or build reth, we just use it in some integration tests. let's remove these. you should also be able to use a smaller image, i suspect.

https://github.com/foundry-rs/foundry/blob/master/Dockerfile#L28

Reply:
Apologies, I got the dependency part from the `reth` docs where it states: `To use Reth, you must first install Geth`. I can make a commit to revert the geth install. I can switch it to a smaller image as well. I mainly kept the `ubuntu` image as its got everything needed for debugging/testing if needed.

Reply:
We should make the readme better then, it's only to run integration tests. Let's indeed switch to smaller image and w/o the geth stuff. People can make debug images separately, let's host a production image.

Reply:
I've updated the image now, it should be ~50MB at this point and there's no geth anymore.